trace/cloud-trace-demo-app/setup.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+################## Set up service a ###########################
+
+echo "Creating service a"
+kubectl apply -f app/demo-service-a.yaml
+
+################## Set up service b ###########################
+echo "Fetching the external IP of service a"
+endpoint=""
+for run in {1..20}
+do
+  sleep 5
+  endpoint=`kubectl get svc cloud-trace-demo-a -ojsonpath='{.status.loadBalancer.ingress[0].ip}'`
+  if [[ "$endpoint" != "" ]]; then
+    break
+  fi
+done
+
+if [[ "$endpoint" == "" ]]; then
+  echo "Unable to get external IP for service cloud-trace-demo-a"
+  exit 1
+fi
+
+echo "Passing external IP for the first service ${endpoint} to the second service template"
+sed "s/{{ endpoint }}/${endpoint}/g" app/demo-service-b.yaml.template > app/demo-service-b.yaml
+kubectl apply -f app/demo-service-b.yaml
+#rm app/demo-service-a.yaml
```

Comment:
This should either be removed, uncommented, or commented with instructions to the end user as to why it's commented out and what they should do with it

Reply:
Removed.